File: Odd or Even.py
import os

def no():
    print("\t\tOdd or Even Number")
    print("\t\t==================")
    x=int(input("Enter Any Number: "))
    if x%2==0:
        print("Even Number")
    else:
        print("Odd Number")
    a=int(input("Enter 1 to start again "))
    if a==1:
        os.system('cls')
        no()
no()

File: Odd_or_Even.java
import java.util.Scanner;

public class Odd_or_Even {
    public static void main(String[] args){
        System.out.println("Enter Number: ");
        Scanner in = new Scanner(System.in);
        int num = in.nextInt();
        if (num%2==0)
                System.out.println("The number "+num+" is Even");
        else
                System.out.println("The number "+num+" is Odd");
    }
}